Tests/SentryTests/SentrySDKIntegrationTestsBase.swift

```diff
@@ -53,15 +53,15 @@ class SentrySDKIntegrationTestsBase: XCTestCase {
         try callback(capture?.event, capture?.scope, capture?.additionalEnvelopeItems)
     }
     
-    func assertCrashEventWithScope(_ callback: (Event?, Scope?) -> Void) {
+    func assertCrashEventWithScope(_ callback: (Event?, Scope?) throws -> Void) throws {
```

Comment:
`m`: See my comment in `Tests/SentryTests/SentryCrash/SentryCrashReportSinkTests.swift` to consider `rethrows` instead of `throws`